presto-hive/src/test/sql/create-test-hive13.sql

```diff
@@ -82,7 +82,7 @@ SELECT * FROM presto_test_types_textfile
 ;
 
 
--- Parquet is missing TIMESTAMP and BINARY, and for some reason
```

Comment:
I have reverted changes in script for 0.13 and created new one for 0.14 - it is copy of 0.13 with timestamp column in Parquet table.

Tested with Hive 0.14.